src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -29,12 +40,54 @@ type Run = {
   error_messages: string | undefined;
 };
 
+type LogsModalProps = {
+  modalTitle: string;
+  logsArray: LogInfo[];
+};
+
+type TaskDataModalProps = {
+  taskDataTitle: string;
+  taskJson: {} | undefined;
```

Comment:
I think that this type is saying that `taskJson` must either be an empty object, or `undefined`. Did you maybe mean `object` instead of `{}`?

Reply:
Ignore this comment! I'm wrong. `{}` is any object that's not `null` or `undefined`.